dozer-sql/src/pipeline/builder.rs

```diff
@@ -690,3 +705,44 @@ mod tests {
         assert_eq!(output_keys, expected_keys);
     }
 }
+
+#[test]
+fn test_missing_into_clause() {
+    let sql = r#"
+    SELECT
+        a.name as "Genre",
+        SUM(amount) as "Gross Revenue(in $)"
+    INTO gross_revenue_stats
+    FROM
+    (
```

Comment:
The check might be triggering on this inner query instead of the outer one. I would suggest splitting this test into multiple simpler ones which each test one edge case. For example:

```sql
SELECT a INTO c FROM b
```
**should not** trigger an error

```sql
SELECT a FROM b
```

**should** trigger an error

```sql
SELECT a INTO c FROM (SELECT a from b)
```

**should not** trigger an error

```sql
SELECT a FROM (SELECT a FROM b)
```

**should** trigger an error